<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_image"/>
    <addaction name="actionSaveImage"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action4PointsRubberRect"/>
    <addaction name="actionProjectiveTransform"/>
    <addaction name="action4PointsWithLines"/>
    <addaction name="actionPixelRuler"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuAddSplitter">
     <property name="title">
      <string>Add splitter</string>
     </property>
     <addaction name="actionVertical"/>
     <addaction name="actionHorizontal"/>
    </widget>
    <addaction name="menuAddSplitter"/>
    <addaction name="actionAddDockWidget"/>
    <addaction name="action4Windows"/>
    <addaction name="action2Windows"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSimultaneousScroll"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
    <widget class="QMenu" name="menuColorSpace">
     <property name="title">
      <string>Color space</string>
     </property>
     <addaction name="actionRGB"/>
     <addaction name="actionHSV"/>
     <addaction name="actionHSI"/>
    </widget>
    <addaction name="menuColorSpace"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuSettings"/>
   <addaction name="menuFilters"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_image">
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="text">
    <string>Vertical</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="text">
    <string>Horizontal</string>
   </property>
  </action>
  <action name="actionAddDockWidget">
   <property name="text">
    <string>Add Dock Widget</string>
   </property>
  </action>
  <action name="action4Windows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>4 Windows</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Save image</string>
   </property>
  </action>
  <action name="actionSimultaneousScroll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simultaneous scroll</string>
   </property>
  </action>
  <action name="actionRGB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="actionHSV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HSV</string>
   </property>
  </action>
  <action name="actionHSI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HSI</string>
   </property>
  </action>
  <action name="action2Windows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2 Windows</string>
   </property>
  </action>
  <action name="action4PointsRubberRect">
   <property name="text">
    <string>4 Points Rubber Rect</string>
   </property>
  </action>
  <action name="actionProjectiveTransform">
   <property name="text">
    <string>Projective Transform</string>
   </property>
  </action>
  <action name="action4PointsWithLines">
   <property name="text">
    <string>4 Points with Aux Lines</string>
   </property>
  </action>
  <action name="actionPixelRuler">
   <property name="text">
    <string>Pixel Ruler</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
